test/sanity/validate-modules/ignore.txt

```diff
@@ -899,6 +899,7 @@ lib/ansible/modules/cloud/vmware/vca_vapp.py E324
 lib/ansible/modules/cloud/vmware/vmware_guest.py E322
 lib/ansible/modules/cloud/vmware/vmware_guest.py E324
 lib/ansible/modules/cloud/vmware/vsphere_copy.py E322
+lib/ansible/modules/cloud/vmware/vmware_guest_vnc.py E326
```

Comment:
Ignore entries should not be added. Instead, fix the underlying issue. In this case, remove the `validate_certs` documentation since it is covered in the docs fragment:

https://github.com/ansible/ansible/blob/5d0ed38e3ad3c4b83bb0c083a8832f8a926da31c/lib/ansible/utils/module_docs_fragments/vmware.py#L37-L42

Reply:
@mattclay I removed those docs..